pyproject.toml

```diff
@@ -365,6 +369,7 @@ doctest_optionflags = [
   "ELLIPSIS",
 ]
 filterwarnings = [
+  "error:_pytest.warning_types.PytestUnknownMarkWarning",
```

Comment:
If we are trying to match the `PytestUnknownMarkWarning` class I think this needs to be `error::_pytest.warning_types.PytestUnknownMarkWarning`?